src/editors/containers/ProblemEditor/data/ReactStateOLXParser.js

```diff
@@ -413,16 +413,18 @@ class ReactStateOLXParser {
             const lowerBoundFloat = Number(numerator) / Number(denominator);
             lowerBoundInt = lowerBoundFloat;
           } else {
-            // these regex replaces remove everything that is not a decimal or positive/negative numer
+            // these regex replaces remove everything that is not a decimal or positive/negative number
             lowerBoundInt = Number(rawLowerBound.replace(/[^0-9-.]/gm, ''));
           }
-          if (rawUpperBound.includes('/')) {
+          if (!rawUpperBound) {
+            upperBoundInt = lowerBoundInt;
```

Comment:
Not blocking, just adding context to the failing codecov check.

It seems codecov isn't considering this line as covered

<img width="1099" height="365" alt="image" src="https://github.com/user-attachments/assets/04bb217d-629a-46c5-b2db-480e76d038c6" />